transport/ws/src/main/java/org/kaazing/gateway/transport/ws/bridge/extensions/idletimeout/IdleTimeoutExtensionFactory.java

```diff
@@ -31,6 +31,12 @@ public String getExtensionName() {
     }
 
     @Override
+    public WebSocketExtension offer(ExtensionHelper extensionHelper, WsResourceAddress address) {
+        // connectors are not sending this extension in handshake request
```

Comment:
Please check with @jfallows that this is ok.  Also, I'm curious if this is how 4.x behaved.  I don't think this was the default behavior of 5.x out of the gateway when it was a transport for socks.